core/templates/dev/head/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.html

```diff
@@ -16,7 +16,8 @@ <h3>Reload a single exploration</h3>
     </div>
     <button type="button"
             class="btn btn-default protractor-test-reload-all-explorations-button"
-            ng-click="$ctrl.reloadAllExplorations()">
+            ng-click="$ctrl.reloadAllExplorations()"
+            ng-disable="$ctrl.reloadingAllExplorationPossible">
```

Comment:
@jameesjohn this should have been ```ng-disabled```? Can you explain to me how ```reloadingAllExplorationPossible``` is supposed to work?

Thanks!

Reply:
@aishwary023 Good catch! Yeah, it should be `ng-disabled`!

it should be` ng-disabled="!$ctrl.reloadingAllExplorationPossible"`